src/DataTable/index.tsx

```diff
@@ -208,6 +208,9 @@ export const DataTable = <T extends Record<string, unknown>>(
   // If the incoming data changes, override the table data
   useEffect(() => {
     setData(data)
+    if(data.length < 1){
```

Comment:
@Emmaalara the table shouldn't do this without the parent project telling it to do so. We should handle this based on an incoming prop - maybe `editing: boolean`

Reply:
We also want this to be possible when the data does exist. So if a table is loaded with current rows, but the user still wants the table to start as editing we'll want to be able to set the table as such